apitools/gen/gen_client_lib.py

```diff
@@ -164,6 +170,73 @@ def WriteInit(self, out):
         printer()
         printer('__path__ = pkgutil.extend_path(__path__, __name__)')
 
+    def WriteIntermediateInit(self, out):
+        """Write a simple __init__.py for an intermediate directory."""
+        printer = self._GetPrinter(out)
+        printer('#!/usr/bin/env python')
+        printer('"""Shared __init__.py for apitools."""')
+        printer()
+        printer('from pkgutil import extend_path')
+        printer('__path__ = extend_path(__path__, __name__)')
+
+    def WriteSetupPy(self, out):
+        """Write a setup.py for upload to PyPI."""
+        printer = self._GetPrinter(out)
+        year = datetime.datetime.now().year
+        printer('# Copyright %s Google Inc. All Rights Reserved.' % year)
+        printer('#')
+        printer('# Licensed under the Apache License, Version 2.0 (the'
+                '"License");')
+        printer('# you may not use this file except in compliance with '
+                'the License.')
+        printer('# You may obtain a copy of the License at')
+        printer('#')
+        printer('#   http://www.apache.org/licenses/LICENSE-2.0')
+        printer('#')
+        printer('# Unless required by applicable law or agreed to in writing, '
+                'software')
+        printer('# distributed under the License is distributed on an "AS IS" '
+                'BASIS,')
+        printer('# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either '
+                'express or implied.')
+        printer('# See the License for the specific language governing '
+                'permissions and')
+        printer('# limitations under the License.')
+        printer()
+        printer('import setuptools')
+        printer('REQUIREMENTS = [')
+        with printer.Indent(indent='    '):
+            # TODO(craigcitro): Have this track apitools' version.
```

Comment:
yeah, i think it's *generally* going to be safe -- but easy enough to fix at some point.